### ISSUE TITLE
Deprecate variable outside an auxillary item? 

### ISSUE THREAD

**Issue by cannin** [sbgn/process-descriptions#218]
Remove the word instance? Also, in oldv2, having the variable outside of the state variable container is deprecated. Should we do the same? It would also simplify greatly the above text that is a bit messy (two boxes, @ is in none of them for example). Finally, the identity of state variables is not clear (we have to keep in mind Andreas' comment on the position of state variables.)

**Comment by cannin** [comment]
I believe it should be deprecated and then we deal with whether constitutes a level versus version change.

**Comment by Adrienrougny** [comment]
I agree. We need to discuss the level vs version issue.

**Comment by cannin** [comment]
Example 
<img width="844" alt="Screenshot 2024-05-30 at 11 38 52 AM" src="https://github.com/sbgn/process-descriptions/assets/790284/4a0b3787-7cf4-4995-9d45-94e37aff04fa">

**Comment by hasanbalci** [comment]
In Section 2.3.2 Glyph: State variable, I replaced 

"Aternatively, the substring identifying the name of the state variable may be displayed using a second label, placed outside of the shape. This is, however, strongly discouraged." by 

"In previous versions of this specification, the substring identifying the name of the state variable was allowed to be displayed using a second label, placed outside of the shape. This alternative approach is now deprecated with this version."

I also replaced "discouraged" by "deprecated" in the caption of Figure 2.3.
